src/test/ui/abi/stack-probes-lto.rs

```diff
@@ -12,6 +12,7 @@
 // ignore-sgx no processes
 // ignore-musl FIXME #31506
 // ignore-pretty
+// ignore-fuchsia Stack overflow caught, but not reported to output
```

Comment:
This comment may be misinterpreted, consider explicitly saying that we don't register an exception handler on Fuchsia